rpm/centos7/scripts/version

```diff
@@ -6,7 +6,7 @@ set -x
 RKE2_POLICYVER=0.3-1 #@TODO: Move this to a more global location.
 RKE2_FALLBACKVER=v1.22.6+rke2r1
 
-TREE_STATE=clean
+TREE_STATE="${TREE_STATE:-clean}"
```

Comment:
TREE_STATE is only used if TAG is not set; could you perhaps just set TAG to the desired value when building?

Reply:
Good point, I've been replacing TREE_STATED value using `sed` on the [RKE2 dev rpms PR](https://github.com/rancher/rke2/pull/2542), so the version script generate a rpm version like this `v1.23.6~rc1~d4807018~rke2r1`. But maybe creating a lightweight tag following the same format would be much easier, let me give it a try.